src/Setup.php

```diff
@@ -415,5 +416,88 @@ public function bcgov_block_theme_wp_headers( $headers ) {
+			'add_new'                  => esc_html__( 'Add new', 'bcgov-block-theme' ),
+			'add_new_item'             => esc_html__( 'Add new Custom Pattern', 'bcgov-block-theme' ),
+			'edit_item'                => esc_html__( 'Edit Custom Pattern', 'bcgov-block-theme' ),
+			'new_item'                 => esc_html__( 'New Custom Pattern', 'bcgov-block-theme' ),
+			'view_item'                => esc_html__( 'View Custom Pattern', 'bcgov-block-theme' ),
+			'view_items'               => esc_html__( 'View Custom Patterns', 'bcgov-block-theme' ),
+			'search_items'             => esc_html__( 'Search Custom Patterns', 'bcgov-block-theme' ),
+			'not_found'                => esc_html__( 'No Custom Patterns found', 'bcgov-block-theme' ),
+			'not_found_in_trash'       => esc_html__( 'No Custom Patterns found in trash', 'bcgov-block-theme' ),
+			'parent'                   => esc_html__( 'Parent Custom Pattern:', 'bcgov-block-theme' ),
+			'featured_image'           => esc_html__( 'Featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'set_featured_image'       => esc_html__( 'Set featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'remove_featured_image'    => esc_html__( 'Remove featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'use_featured_image'       => esc_html__( 'Use as featured image for this Custom Pattern', 'bcgov-block-theme' ),
+			'archives'                 => esc_html__( 'Custom Pattern archives', 'bcgov-block-theme' ),
+			'insert_into_item'         => esc_html__( 'Insert into Custom Pattern', 'bcgov-block-theme' ),
+			'uploaded_to_this_item'    => esc_html__( 'Upload to this Custom Pattern', 'bcgov-block-theme' ),
+			'filter_items_list'        => esc_html__( 'Filter Custom Patterns list', 'bcgov-block-theme' ),
+			'items_list_navigation'    => esc_html__( 'Custom Patterns list navigation', 'bcgov-block-theme' ),
+			'items_list'               => esc_html__( 'Custom Patterns list', 'bcgov-block-theme' ),
+			'attributes'               => esc_html__( 'Custom Patterns attributes', 'bcgov-block-theme' ),
+			'name_admin_bar'           => esc_html__( 'Custom Pattern', 'bcgov-block-theme' ),
+			'item_published'           => esc_html__( 'Custom Pattern published', 'bcgov-block-theme' ),
+			'item_published_privately' => esc_html__( 'Custom Pattern published privately.', 'bcgov-block-theme' ),
+			'item_reverted_to_draft'   => esc_html__( 'Custom Pattern reverted to draft.', 'bcgov-block-theme' ),
+			'item_scheduled'           => esc_html__( 'Custom Pattern scheduled', 'bcgov-block-theme' ),
+			'item_updated'             => esc_html__( 'Custom Pattern updated.', 'bcgov-block-theme' ),
+			'parent_item_colon'        => esc_html__( 'Parent Custom Pattern:', 'bcgov-block-theme' ),
+		];
+
+		$args = [
+			'label'                 => esc_html__( 'Custom Patterns', 'bcgov-block-theme' ),
+			'labels'                => $labels,
+			'description'           => '',
+			'public'                => true,
```

Comment:
are we sure we want it public, can't remember if this removes it from the admin, What ever we need to do to make this cpt not accessible in the public web pages, we should do (only in admin)

Reply:
'publicly_queryable'    => false, <-- sets it to not accessible as a page/post. I've set this.